addons/stock/models/procurement.py

```diff
@@ -154,7 +154,7 @@ def _get_stock_move_values(self):
         qty_left = max(self.product_qty - qty_done, 0)
         return {
             'name': self.name[:2000],
-            'company_id': self.rule_id.company_id.id or self.rule_id.location_src_id.company_id.id or self.rule_id.location_id.company_id.id or self.company_id.id,
+            'company_id': self.rule_id.company_ id.id or self.rule_id.location_src_id.company_id.id or self.rule_id.location_id.company_id.id or self.company_id.id,
```

Comment:
here is an error

Reply:
I know.  We can not do this branch in stable anyways, so I am doing a new one.  https://github.com/odoo/odoo/pull/25401